appmail/models.py

```diff
@@ -199,10 +201,11 @@ def _validate_subject(self):
     def render_body(self, context, content_type=CONTENT_TYPE_PLAIN, processors=CONTEXT_PROCESSORS):
         """Render email body in plain text or HTML format."""
         assert content_type in EmailTemplate.CONTENT_TYPES, _lazy("Invalid content type.")
-        ctx = Context(helpers.patch_context(context, processors))
         if content_type == EmailTemplate.CONTENT_TYPE_PLAIN:
-            return Template(self.body_text).render(ctx)
+            ctx = self.patch_safe_context(context, processors)
+            return unescape(Template(self.body_text).render(ctx))
```

Comment:
1. Last commit message says "mark context safe instead of unescaping it".

Is this `unescape()` still needed?

2. I'm not sure that `patch_safe_context` is a clear name and I'm not sure why this should be a class method. I think it's better to move it to a helper function and name `mark_context_safe()` + remove processors from the function.

```
def mark_context_safe(context):
    return {key: mark_safe(value) for key, value in context.items()}
```

and then use it within the class. It keeps `Context(...)` call on the same level of function calls — on `render_body()`/`render_subject()` level. It will help avoid confusion in the future — why some `ctx` is built from `Context()` function and other ctx is returned by function `patch_safe_context()`.